AppCenter/AppCenterTests/MSLoggerTests.m

```diff
@@ -48,4 +48,17 @@ - (void)testSetCurrentLoglevelWorks {
   XCTAssertTrue([MSLogger currentLogLevel] == MSLogLevelWarning);
 }
 
+- (void)testLoglevelNoneDoesNotLogMessages {
+
+  // If
+  MSLogMessageProvider messageProvider = ^() {
+    XCTFail(@"Log shouldn't be printed.");
+    return @"";
+  };
+
+  // When
```

Comment:
No `// Then`?

Reply:
The line 55 is kind of `// Then` but basically it is testing that the operation doesn't do anything.